### PR TITLE
Dread: Revise logic for Spin Boost Tower

### PR DESCRIPTION
**Pull request by XenoWars**
- Added some speedbooster methods for climbing to various points in the room
- Added a speedbooster method of getting the power bomb tank
- Properly accounted for the ability to get the energy part item at the grapple block
- Removed the "Energy Part Platform" node as I thought it was pretty redundant
- Added advanced HBJ for Left Entrance item

### CODE REVIEW

File: randovania/games/dread/json_data/Ghavoran.txt [hunk 1]
```diff
@@ -1583,25 +1625,16 @@ Extra - asset_id: collision_camera_021
 > Ledge Below PB Tank; Heals? False
   * Layers: default
   > Door to Above Pulse Radar
-      Simple IBJ or Use Spin Boost
+      Movement (Beginner) or Simple IBJ or Use Spin Boost
```
Comment (kirbymastah) [inline randovania/games/dread/json_data/Ghavoran.txt:1628]: Not sure what the "Movement (Beginner)" method is here - is it a slide jump? Could you add a comment clarifying this?

**Review by kirbymastah — CHANGES_REQUESTED** [review]
Just one quick comment

File: randovania/games/dread/json_data/Ghavoran.txt (hunk 1 above)
Comment (XenoWars) [reply randovania/games/dread/json_data/Ghavoran.txt:1628]: you can jump to the edge, but it's not a jump that is very simple. the jump doesnt even look doable, but it is

Comment (XenoWars) [reply randovania/games/dread/json_data/Ghavoran.txt:1628]: i also tested this a while ago back when i first did the revision. i could try again and see if it should just be a trivial connection

**Review by kirbymastah — APPROVED** [review]

**Pull request merged**